src/shared/utils/arrowUpAutoFill.ts

```diff
@@ -0,0 +1,28 @@
+export const arrowUpAutoFill = () => {
+    window.onload = () => {
+        const $inputBox = document.getElementById("prompt-textarea");
+        let $lastMessage: string | null = "";
+
+        const updateDivs = () => {
+            const $divs = document.querySelectorAll(
+                'div[data-message-author-role="user"]',
```

Comment:
We can query the element we need with just a CSS [combinator selector](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators). We also have a variable that stores the CSS selector for all the message bubbles, defined [here](https://github.com/joseph-w-valdez/chatGPTStyler/blob/93f4183cfc59a1d697db1bbad28b0a9ab766c333/src/shared/utils/stylingFunctions.ts#L3). We should export that and use it as a part of the combinator selector. It would look something like
```document.querySelector(`:nth-last-child(2 of ${messageBubbles}) [data-message-author-role="user"] > div') ```
<details><summary>Deep Dive</summary>
<p>

<details><summary>More about our `messageBubbles` variable</summary>
<p>

The selector we use for the message bubbles is `[data-testid^="conversation-turn-"]`.
By using `^=` we are saying  "all element that have a `data-testid` attribute with a value that _**starts with**_ `"conversation-turn-"`.
Although many elements on Chat GPT have `data-testid` attributes, we are confident that only the elements we consider to be "message bubbles" will be targeted by our selector. 

</p>
</details> 

```:nth-last-child(2 of ${messageBubbles})``` selects the 2nd-last message bubble.
<details><summary>:nth-last-child() Details</summary>
<p>

uses [:nth-last-child()](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-last-child) with the `of <selector>` syntax. By passing a "step count" of 2 and our message bubbles selector (`messageBubbles`) as the selector argument, we can select the 2nd-last (the "step count") element that matches that selector (our message bubbles).  to create a selector for the second to last message bubble.

</p>
</details> 
By using combinators in between our selectors, we end up getting the `div` that has the text content of the user's latest message.
<details><summary>Combinator Details</summary>
<p>

2 combinators, the [descendant combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator) and the [child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator).

This says to select all `div` elements that are children of all elements with the attribute `data-message-author-role="user"` that have ancestors that are elements that match the first selector (the 2nd-last message bubble)

</p>
</details> 

</p>
</details>